model/gen-ai/registry.yaml

```
@@ -211,6 +211,11 @@ groups:
               brief: 'Output tokens (completion, response, etc.)'
         brief: The type of token being counted.
         examples: ['input', 'output']
+      - id: gen_ai.conversation.id
+        stability: development
+        type: string
+        brief: The unique identifier of the conversation, use the same value across requests to continue the same conversation.
```

```suggestion
        brief: The unique identifier for a conversation, used to correlate related requests within the same context.
```
What do you think about this change to the description?